src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -2939,6 +2953,11 @@ subroutine gsi_fv3ncdf_read_ens_parallel_over_ens(filenamein,fv3filenamegin, &
           allocate(varname_files(1))
           varname_files = (/'ref_f3d'/)
        end if
+       if( present(fed) )then            ! phyvars: fed 
```

Comment:
when both dbz and fed exist,  what will happen?

Reply:
@TingLei-NOAA 
As we discussed before, dbz and fed are read by two seperate calls, so this is not an issue. However, the code will be revised to handle the case when both dbz and fed are presented. It will like: 

          if( present(dbz) .and. present(fed) )then ! phyvars: dbz,fed
            if(ivar == 1) dbz = hwork
            if(ivar == 2) fed = hwork
          elseif( present(dbz) )then            ! phyvars: dbz
            dbz = hwork
          elseif( present(fed) )then            ! phyvars: fed 
            fed = hwork
          end if

Reply:
@TingLei-NOAA 
tested. it works when both dbz and fed are true.